### ISSUE TITLE
Archive and RSS not being updated

### ISSUE THREAD

**Issue by aleprovencio** [kale/hackernewsletter.com#2]
Hello,
the `Past Issues` and `RSS feed` are not being updated with the last issues. 

Sorry if this is not the correct place to warn about this, could not find a better one.

**Comment by luv2code** [comment]
It would be cool if this was fixed.

**Comment by trish332** [comment]
Indeed. I really appreciate RSS for the newsletter and use it regularly.

**Comment by luv2code** [comment]
@kale could we get a response on this? Maybe the RSS campaign got paused by accident?

**Comment by aleprovencio** [comment]
@trish332, @luv2code 
maybe you guys have other means of contacting him, like twitter maybe?

**Comment by aleprovencio** [comment]
Seems to be working again :)

**Comment by plaza-developers** [comment]
What is the feed url?
@aleprovencio

**Comment by aleprovencio** [comment]
Sorry, not using anymore in favor of https://www.daemonology.net/hn-daily/
